### PR TITLE
Mention `Content-Disposition` header for icons endpoint in docs

### PR DESCRIPTION
**Pull request by stevenwilkin**
Per #287

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

File: docs/rest.md [hunk 1]
```diff
@@ -538,7 +538,8 @@ Notes: user facing implementations in text form must show this data using yaml.
 
 * Description: Get an icon from a snap installed on the system. The
   response will be the raw contents of the icon file; the content-type
-  will be set accordingly.
+  will be set accordingly and the content-disposition will specify the
```
Comment (mvo5) [inline docs/rest.md]: (nitpick^2) Maybe "content-disposition" -> "content-disposition header" ?

Comment (stevenwilkin) [inline docs/rest.md]: Good call!

**Comment by mvo5** [comment]
Thanks! Looks good, one question/suggestion but feel free to ignore.

**Comment by stevenwilkin** [comment]
Cheers @mvo5, I've updated the branch to incorporate your feedback.

**Comment by chipaca** [comment]
:+1:

**Pull request merged**